### PR TITLE
must-gather: add listomapvals command for ceph rados

### PR DESCRIPTION
**Pull request by agarwal-mudit**
Signed-off-by: Mudit Agarwal <muagarwa@redhat.com>

### CODE REVIEW

File: must-gather/collection-scripts/gather_ceph_resources [hunk 1]
```diff
@@ -179,15 +179,23 @@ for ns in $namespaces; do
...
             # Get the omap details of each snap object
             for snapobj in $snapobjs; do
                 { printf "Name of snap object: %s\n" "${snapobj}" >> "${COMMAND_OUTPUT_FILE}"; }
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rados getomapval csi.snaps.default ${snapobj} --pool=${mdpool} --namespace=csi" >> "${COMMAND_OUTPUT_FILE}"; } >>   "${COMMAND_ERR_OUTPUT_DIR}"/gather-rados-"${snapobj}"-debug.log 2>&1 &
+                # getomapval writes the UUID to a file inside helper pod
+                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rados getomapval csi.snaps.default ${snapobj} --pool=${mdpool} --namespace=csi uuidfile"; }
+                # Get UUID from the file
+                UUID=$(oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "cat uuidfile")
+                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rados listomapvals csi.snap.${UUID} --pool=${mdpool} --namespace=csi" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rados-"${snapobj}"-debug.log 2>&1 &
```
Comment (Madhu-1) [inline must-gather/collection-scripts/gather_ceph_resources:198]: Need to remove the `uuidfile` file once everything is done?

**Review by Madhu-1 — COMMENTED** [review]
LGTM. left a  question

File: must-gather/collection-scripts/gather_ceph_resources (hunk 1 above)
Comment (Madhu-1) [reply must-gather/collection-scripts/gather_ceph_resources:198]: Thanks, @agarwal-mudit as discussed it's inside the helper pod. deleting helper pod will take care of cleanup

**Review by Madhu-1 — APPROVED** [review]

File: must-gather/collection-scripts/gather_ceph_resources (hunk 1 above)
Comment (agarwal-mudit) [reply must-gather/collection-scripts/gather_ceph_resources:198]: Yeah, it is inside the helper pod. Thanks for the review Madhu

**Pull request merged**